Dockerfile

```diff
@@ -42,7 +50,11 @@ RUN curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.33.2/install.sh
     echo 'CI=Y' >> ${HOME}/.bashrc && \
     nvm install ${NODEJS_DEFAULT_VERSION} && \
     chmod -R 777 ${NVM_DIR} && \
-    npm install -g cordova@${CORDOVA_DEFAULT_VERSION}
+    npm install -g cordova@${CORDOVA_DEFAULT_VERSION} && \
+    gem install fastlane@${FASTLANE_DEFAULT_VERSION} && \
```

Comment:
Correct way to specify version in gem command: `gem install fastlane -v ${FASTLANE_DEFAULT_VERSION}`